common/configure/commonreplicatorconfig.go

```diff
@@ -42,3 +47,14 @@ func (r *ReplicatorConfig) GetReplicatorHosts() map[string]string {
 func (r *ReplicatorConfig) GetDefaultAuthoritativeZone() string {
 	return r.DefaultAuthoritativeZone
 }
+
+// GetUseStandalone checks whether a specific deployment is using standalone deployment
+func (r *ReplicatorConfig) GetUseStandalone(deployment string) bool {
```

Comment:
Yea, open source is not using this config. Right now we don't have a clean way to have 'internal only' config(interface is defined in open source) so keep it here for now.